src/components/Menu/Menu.jsx

```diff
@@ -1,13 +1,51 @@
-import styles from "./Menu.module.scss";
+import { useState } from 'react';
+import styles from './Menu.module.scss';
+
 function Menu() {
-  return (
-    <div className={styles.Menu}>
-      Menu
-      <span className={styles.Line1}></span>
-      <span className={styles.Line2}></span>
-      <span className={styles.Line3}></span>
-    </div>
-  );
+    const [menuOpen, setMenuOpen] = useState(false);
+
+    return (
+        <div className={styles.Menu}>
+            {menuOpen ? (
+                <div className={styles.MenuOpen}>
+                    <div
+                        className={styles.MenuLinesOpen}
+                        onClick={() => setMenuOpen(false)}
+                    >
+                        <div className={styles.LinesContainerOpen}>
+                            <div className={styles.LineOpen1}></div>
+                            <div className={styles.LineOpen2}></div>
+                        </div>
+                    </div>
+                    <div>
+                        <div className={styles.MenuHeader}>Meny</div>
+                        <div className={styles.MenuItem}>Om Oss</div>
```

Comment:
And should these be inside a `ul` as `li`s to be consistent with menu building standards? :)